src/snek5000/output/history_points.py

```diff
@@ -57,12 +59,66 @@ def _complete_params_with_default(cls, params):
         )
 
     def load(self):
+        if self._data is None:
+            return self._load_full()
+        else:
+            nb_points = self.coords.shape[0]
+            nb_data_lines_read = len(self._data)
+
+            size_file = self.path_file.stat().st_size
+            print(f"  {size_file = }")
+
+            with open(self.path_file) as file:
+                first_line = file.readline()
+                line_coord = file.readline()
+                for _ in range(nb_points - 1):
+                    file.readline()
+                line_data = file.readline()
+
+            nb_chars_read = (
+                len(first_line)
+                + nb_points * len(line_coord)
+                + nb_data_lines_read * len(line_data)
+            )
```

Comment:
Can we keep track of `nb_chars_read` in a class attribute and then seek to the position at once?

Reply:
I don't think it's necessary. It would be less clear and in some corner cases less robust.